edalize/symbiflow.py

```diff
@@ -207,23 +215,32 @@ def configure_vpr(self):
 
         part = self.tool_options.get('part', None)
         package = self.tool_options.get('package', None)
+        vendor = self.tool_options.get('vendor', None)
 
         assert part is not None, 'Missing required "part" parameter'
         assert package is not None, 'Missing required "package" parameter'
 
-        if 'xc7a' in part:
-            bitstream_device = 'artix7'
-        if 'xc7z' in part:
-            bitstream_device = 'zynq7'
-        if 'xc7k' in part:
-            bitstream_device = 'kintex7'
-
-        partname = part + package
-
-        # a35t are in fact a50t
-        # leave partname with 35 so we access correct DB
-        if part == 'xc7a35t':
-            part = 'xc7a50t'
+        if vendor == 'xilinx':
+            if 'xc7a' in part:
+                bitstream_device = 'artix7'
+            if 'xc7z' in part:
+                bitstream_device = 'zynq7'
+            if 'xc7k' in part:
+                bitstream_device = 'kintex7'
+
+            partname = part + package
+
+            # a35t are in fact a50t
+            # leave partname with 35 so we access correct DB
+            if part == 'xc7a35t':
+                part = 'xc7a50t'
+            device_suffix = 'test'
+            toolchain_prefix = 'symbiflow_'
+        elif vendor == 'quicklogic':
+            partname = package
+            device_suffix = 'wlcsp'
+            bitstream_device = part + "_" + device_suffix
+            toolchain_prefix = ''
```

Comment:
why is toolchain prefix an empty string?

Reply:
Quicklogic toolchain v1.1.0 (downloaded from: https://github.com/QuickLogic-Corp/quicklogic-fpga-toolchain/releases/download/v1.1.0/Symbiflow_v1.1.0.gz.run) do not have any prefix before toolchain executables (e.g. executable is named ``synth``). If it will change in future releases this needs to be adjusted.

Reply:
Ok, probably worth adding a small comment about it

Reply:
done